_docs/layout/navigation.md

```diff
@@ -7,33 +7,42 @@ preview-image: layout/preview-images/navigation.svg
 resource: true
 ---
 
-Navigation establishes where the user is in an application and provides a path
-to complete tasks. Consistent navigation patterns allow the user to complete
-tasks more efficiently and confidently. Rackspace applications may utilize a
-combination of eyebrow navigation, either a top or side navigation, and
-breadcrumbs.
+Navigation establishes a user's location within an application and
+shows the path forward to accomplish a goal. Consistent navigation
+patterns increase user confidence and efficiency. You can use a
+combination of eyebrow navigation, either top or side navigation, and
```

Comment:
So, I wonder if we should remove 'eyebrow navigation'? There will be users outside of designers that will be accessing this content. It's not necessarily colloquial, but it's not exactly common either. @palindromebart - can we think about this?